tools/sim/start_carla.sh

```diff
@@ -15,5 +15,4 @@ if [ ! -d carla ]; then
 fi
 
 cd carla
-./CarlaUE4.sh Town04
-
+./CarlaUE4.sh 
```

Comment:
Why change this?

Reply:
It's set via the client, Town04 changes nothing, town gets loaded in bridge.py in line 89.